### ISSUE TITLE
Arch Linux Not connecting while Phone does

### ISSUE THREAD

**Issue by backspacestud** [notthebee/ansible-easy-vpn#115]
I have gone through the setup and verified that it works on my mobile phone (it connected and worked fine) same with the web portal also working. Yet when setting it up on my arch linux desktop I tried to use the CLI method of Wireguard to no avail possibly causing issues with UFW along the way (yet i have deleted UFW and made sure to disable the systemd service i created). I then moved on to create it through the network connections system settings module in KDE using the import VPN configuration option and I am able to connect to the service yet have no internet access. If anyone has any ideas on how to fix this that would be greatly appreciated.

**Environment:**
- Client OS: Arch linux
- VPS OS: Ubuntu 20.04
- VPS Provider: RackNerd

**Comment by notthebee** [comment]
Are both devices on the same Internet connection?

**Comment by backspacestud** [comment]
Yes, would that cause an issue?

**Comment by notthebee** [comment]
Nope, not at all. Are you using the same 'client' file for both your phone and your Arch Linux box?

**Comment by backspacestud** [comment]
No, i created one for my phone and one for my desktop

**Comment by notthebee** [comment]
Ok, that's good. What is the exact command you're using to connect to the VPN from the CLI?

**Comment by backspacestud** [comment]
I gave up on using the CLI and instead used the Connections Tab in KDE system settings because it had an option to import a VPN connection with a config file and thought it would be easiest.

**Comment by notthebee** [comment]
I see.

If you're absolutely sure that it works on your phone (e.g. verified that the public IP address matches with the VPS host using something like https://whoer.net ), then there might be some kind of a firewall issue on the Linux side. At the same time, I don't think Arch Linux comes with any kind of firewall rules by default, so I'm kind of clueless here 😁

Is there any chance you can post Wireguard client logs? I'm not sure where to find them on Linux, but it would be helpful

**Comment by backspacestud** [comment]
I have no firewall and still have the issue, and I am unsure where to find the wireguard logs. I tried to get information that may be related? but not exactly sure what is sensitive or not so i just blurred a lot of things. 
![Screenshot_20230103_001148](https://user-images.githubusercontent.com/101374988/210321243-7aed5aca-628e-443a-944e-d7ef8ce55bbd.png)
![Screenshot_20230103_000413](https://user-images.githubusercontent.com/101374988/210321324-08628de7-5533-4546-a435-4bc30c17996d.png)

**Comment by notthebee** [comment]
Does the client show in the wg-easy WebUI at all when you're connected? Usually with the client connected the WebUI would show some stats, like the latest handshake, traffic, etc.

**Comment by backspacestud** [comment]
oddly enoyghg it does not say it ever connects but does have little graphical pings. Its alittle hard to screenshot because i can only view it in real time on my phone because when i connect to the VPN on my desktop it again has no internet access
![Screenshot_20230103_003345](https://user-images.githubusercontent.com/101374988/210322937-4351e7ac-5f3e-4440-98ad-1a16cf2aaaf5.png)
![Screenshot_20230103_003355](https://user-images.githubusercontent.com/101374988/210322938-5168db63-874f-41e2-9a20-ed43e3ea433b.png)

**Comment by backspacestud** [comment]
Im beginning to think its just a weird issue deep within my computer messing with connecting to networks due to previous networking experiments, I have a track record of running into random obscure issues and at this point its getting late. Tomorrow i might just do a complete reinstall of arch or something crazy to get it to work. Or more likely try to use the CLI again. I will keep trying to post updates and thank you for all your help so far. :)

**Comment by notthebee** [comment]
One thing you might want to check – is the Internet connection missing completely or do you just have no DNS?
Can you ping 1.1.1.1?

**Comment by backspacestud** [comment]
I fixed it! i just learned to not trust the KDE connections panel and instead just do it through the CLI. It was super simple and the error i had before `/usr/bin/wg-quick: line 32: resolvconf:` command not found was simply fixed by re-installing resolvconf. After that it was as simple as moving the conf file into /etc/wireguard and using the command: 

`sudo wg-quick up wg0` 

(replace wg0 with whatever the name of the conf file is excluding the .conf at the end).

for any linux users just use the cli its super simple and if you want to set up a kill switch use a firewall (there are plenty of tutorials online) and if you want to setup a on startup use a systemd service (again there are plenty of tutorials online)!

here are two tutorials by IVPN, I just have not tried them myself yet but they seem like they would work: 
[Killswitch](https://www.ivpn.net/knowledgebase/linux/linux-wireguard-kill-switch/)
[Autostart](https://www.ivpn.net/knowledgebase/linux/linux-autostart-wireguard-in-systemd/)

Sorry for the off topic rant and the random issue, but thank you for all the help. I appreciate this script and your videos!
